src/components/item/sharing/membershipTable/GuestItemMembershipTableRow.tsx

```diff
@@ -50,7 +39,7 @@ const GuestItemMembershipTableRow = ({
       <StyledTableRow data-cy={buildItemMembershipRowId(data.id)} key={data.id}>
         <TableCell>
           <Typography
-            sx={{ color: isDisabled ? DISABLED_COLOR : undefined }}
+            sx={{ color: isDisabled ? DEFAULT_TEXT_DISABLED_COLOR : undefined }}
```

Comment:
Maybe you can set `disabled` prop directly on typography ? or use `color="text.disabled"`